tests/FormDataFactory/DemoFormDataFactoryTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Website\Tests\FormDataFactory;
+
+use DateTimeImmutable;
+use Ramsey\Uuid\Uuid;
+use Rector\Website\Entity\RectorRun;
+use Rector\Website\FormDataFactory\DemoFormDataFactory;
+use Rector\Website\GetRectorKernel;
+use Symplify\PackageBuilder\Tests\AbstractKernelTestCase;
+
+/**
+ * @see \Rector\Website\FormDataFactory\DemoFormDataFactory
+ */
+final class DemoFormDataFactoryTest extends AbstractKernelTestCase
+{
+    /**
+     * @var DemoFormDataFactory
+     */
+    private $demoFormDataFactory;
+
+    protected function setUp(): void
+    {
+        self::bootKernel(GetRectorKernel::class);
```

Comment:
`$this->bootKernel()` should be used instead, `bootKernel()` method is not static. PHPStorm is reporting this to me as bug.

Reply:
I see that symfony's `KernelTestCase` is static but in symplify `AbstractKernelTestCase` it is not static

Reply:
Yes indeed :+1: Thanks